kubernetes.yml

```diff
@@ -0,0 +1,45 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: windows-pod
+  labels:
+    name: windows-pod
+spec:
+  containers:
+    - name: windows
+      image: dockurr/windows
+      ports:
+        - containerPort: 8006
+        - containerPort: 3389
+          protocol: TCP
+      resources:
+        limits:
+          devices.kubevirt.io/kvm: 1
```

Comment:
@wangwei10061 I would remove the `kubevirt.io` part because not all clusters support it by default.

Reply:
> @wangwei10061 I would remove the `kubevirt.io` part because not all clusters support it by default.

I agree this should be replaced by 
```
volumeMounts:
        - mountPath: /dev/kvm
          name: dev-kvm

....
volumes:
    - name: dev-kvm
      hostPath:
        path: /dev/kvm
```

Otherwise the dependencies in the cluster are Kubevirt install for a simple mount. 

````
 export VERSION=$(curl -s https://storage.googleapis.com/kubevirt-prow/release/kubevirt/kubevirt/stable.txt)
  kubectl create -f https://github.com/kubevirt/kubevirt/releases/download/${VERSION}/kubevirt-operator.yaml
  kubectl create -f https://github.com/kubevirt/kubevirt/releases/download/${VERSION}/kubevirt-cr.yaml
```

The idea of `devices.kubevirt.io/kvm: 1` is that doesn't require NET_ADMIN but dockur is already using it in the containers so I don't see the point. 

Tested in qbo kubernetes